kotlinlib/src/mill/kotlinlib/KotlinModule.scala

```diff
@@ -402,6 +404,10 @@ trait KotlinModule extends JavaModule { outer =>
     )
   }
 
+  def kotlinPrecompiledClasses: Task[Seq[PathRef]] = Task {
```

Comment:
What are these? Who is supposed to pre-compile them?

Reply:
I've removed that, the compiled classpath should work now properly, it was an early effort to make things work quickly without cyclic calls and stack overflow, but I separated the kspClasspath from the compile classpath now

Reply:
I still see `def kotlinPrecompiledClasses` and it's use sites.